config/default.js

```diff
@@ -60,6 +60,11 @@ module.exports = {
     privateKey: "<privateKey>",
   },
 
+  botToken: {
+    publicKey: "<publicKey>",
+    privateKey: "<privateKey>",
+  },
+
```

Comment:
Are these new keys that are being added from env?
Then we will have to add it in custom-environment-variables.js file

Reply:
Added